ext/debian/puppet.conf

```diff
@@ -0,0 +1 @@
+../../conf/puppet.conf
```

Comment:
We pulled symlinks from the acceptance repo in: https://github.com/puppetlabs/puppet/commit/959e0cc8559afb879a10d62425788e729b5b2503 They were also causing problems for @hlindberg in Eclipse, just by being in the repository, as I recall.  I don't know if their being in ext makes either point less of an issue.